### ISSUE TITLE
`tox` command not running unittests

### ISSUE THREAD

**Issue by peterschutt** [MikeWooster/api-client#63]
I'm fairly new to using tox in my own projects so I'd have to do some digging to work out a fix, but if I insert an explicitly failing test, the `tox` command still returns successes. I think it is just building each environment but not running the tests on them.

**To Reproduce**
See PR #62 which I've made as demonstration that the CI tests pass.

**Expected behavior**
CI should fail.

**Comment by peterschutt** [comment]
I'm pretty sure I struggled to work this out with my own libs and just ended up putting the `extras=...` and the `commands=...` into the `[testenv]` section. This is the tox.ini from my template library project:

```
[tox]
envlist=lint,py38,py39
isolated_build=true

[testenv]
description=Runs pytest tests in each of the py environs
extras=tests
commands=
    pytest --cov={envsitepackagesdir}/penguin

[testenv:lint]
description=Lint project files
extras=lint
commands =
    isort --check --diff .
    black --check --diff .
    flake8 src/ tests/
    mypy src tests

```
I don't know if that is good or bad practice but it ends up running linting on the environment python and unittests on the two specified python versions.

**Comment by MikeWooster** [comment]
Many thanks.  I'll take a look and see if I can fix this and #61

**Comment by MikeWooster** [comment]
fixed in #64
